Models/testmodel1/config.cpp

```diff
@@ -0,0 +1,49 @@
+#include "config.hpp"
+
+//TODO: Consider where these should be initialized
+TempPWMDriver PWMDriver;
+TempDSHOTDriver DSHOTdriver;
```

Comment:
I think the discussion suggested that we initialize these in the module that uses them (Attitude Manager)